presentation-compiler/src/main/dotty/tools/pc/PcInlineValueProvider.scala

```diff
@@ -133,11 +133,8 @@ final class PcInlineValueProvider(
     rhsLines match
       case h :: Nil => rhs
       case h :: t =>
-        val noPrefixH = h.stripPrefix(refIndent)
-        if noPrefixH.startsWith("{") then
-          noPrefixH ++ t.map(refIndent ++ _.stripPrefix(defIndent)).mkString("\n","\n", "")
-        else
-          (("  " ++ h) :: t).map(refIndent ++ _.stripPrefix(defIndent)).mkString("\n", "\n", "")
+        val header = if h.startsWith("{") then h else "\n" ++ refIndent ++ "  " ++ h
```

Comment:
Nice find!

Although, I think that the `h.startsWith("{")` is wrong too :see_no_evil:
The `else` branch should most likely only be performed if the **start line number of the rhs** is different than the **end line number of the definition**.